src/utils/validateStrings.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Checks if an empty string of any length has been passed in
+ * @param {string} itemName
+ * @returns {boolean}
+ */
+export const isEmpty = (itemName) =>
+	typeof itemName !== 'string' || itemName.trim().length === 0;
+
+export const isDuplicate = (itemName, data) =>
+	data.some(({ name }) => cleanup(name) === cleanup(itemName));
```

Comment:
Nice job making this very readable and modular (especially when imported to AddItem component)